test/Analysis/ctu-main_usr.cpp

```diff
@@ -1,12 +1,12 @@
-// RUN: mkdir -p %T/xtudir
-// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -std=c++11 -emit-pch -o %T/xtudir/xtu-other.cpp.ast %S/Inputs/xtu-other.cpp
-// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -std=c++11 -emit-pch -o %T/xtudir/xtu-chain.cpp.ast %S/Inputs/xtu-chain.cpp
-// RUN: cp %S/Inputs/externalFnMap_usr.txt %T/xtudir/externalFnMap.txt
-// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -fsyntax-only -analyze -analyzer-checker=core,debug.ExprInspection -analyzer-config xtu-dir=%T/xtudir -analyzer-config use-usr=true -analyzer-config reanalyze-xtu-visited=true -std=c++11 -verify %s
-// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -fsyntax-only -analyze -analyzer-checker=core,debug.ExprInspection -analyzer-config xtu-dir=%T/xtudir -analyzer-config use-usr=true -analyzer-config reanalyze-xtu-visited=true -std=c++11 -analyzer-display-ctu-progress 2>&1 %s | FileCheck %s
-
-// CHECK: ANALYZE (CTU loaded AST for source file): {{.*}}/xtu-other.cpp
-// CHECK: ANALYZE (CTU loaded AST for source file): {{.*}}/xtu-chain.cpp
+// RUN: mkdir -p %T/ctudir
+// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -std=c++11 -emit-pch -o %T/ctudir/ctu-other.cpp.ast %S/Inputs/ctu-other.cpp
+// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -std=c++11 -emit-pch -o %T/ctudir/ctu-chain.cpp.ast %S/Inputs/ctu-chain.cpp
+// RUN: cp %S/Inputs/externalFnMap_usr.txt %T/ctudir/externalFnMap.txt
+// RUN: %clang_cc1 -triple x86_64-pc-linux-gnu -fsyntax-only -analyze -analyzer-checker=core,debug.ExprInspection -analyzer-config ctu-dir=%T/ctudir -analyzer-config use-usr=true -analyzer-config reanalyze-ctu-visited=true -std=c++11 -verify %s
```

Comment:
Same as above, command line could be simplified. Similarily we only support USR now, so `-analyzer-config use-usr=true` can be removed as well. It looks like I forgot to update the tests after those changes.

Reply:
The redundant `-analyzer-config use-usr=true -analyzer-config reanalyze-ctu-visited=true` were not removed here and bellow.